Source/WebCore/animation/ViewTimeline.h

```diff
@@ -66,9 +67,10 @@ class ViewTimeline final : public ScrollTimeline {
 
     RenderBox* sourceScrollerRenderer() const;
     Element* source() const override;
+    TimelineRange defaultRange() const override;
 
 private:
-    ScrollTimeline::Data computeTimelineData(const TimelineRange& = { }) const override;
+    ScrollTimeline::Data computeTimelineData(const TimelineRange&) const override;
```

Comment:
nit: those methods can probably be final

Reply:
Fixed.